File: Backend/Internal/Entities/Achievement.cs
namespace Hackaton_DW_2024.Internal.Entities;

public class Achievement
{
    public int Id { get; set; }
    public int Score { get; set; }
    public int TeamSize {get; set; }
    public int UserId { get; set; }
    public string FilePath { get; set; }
}

File: frontend/src/entities/achievement/Achievement.ts
export interface Achievement{
    id: number | string;
    score: number | string;
    withTeam: boolean;
    userId: number | string;
    filePath: string;
    resultId: number;
}

export interface AchConnected{
    id: number;
    eventId: number;
    resultId: number;
    withTeam: boolean;
}

export interface AchCustom{
    id: number;
    title: string;
    date: string;
    statusId: number;
    resultId: number;
    withTeam: boolean;
}